spring-session/src/main/java/org/springframework/session/web/http/OnCommittedResponseWrapper.java

```diff
@@ -529,6 +530,14 @@ public void println(double d) throws IOException {
 			this.delegate.println(d);
 		}
 
+		public boolean isReady() {
```

Comment:
Right but you will notice that none of the other repository implementations have web dependencies in them. Doing this will cause [package tangles](http://structure101.com/blog/2008/12/software-erosion-and-package-tangles/).

I really think we need to consider global namespace as its own thing as a PR to add new repository should not contain web dependencies nor should it change anything out of its package structure.